solver/llbsolver/solver.go

```diff
@@ -314,39 +325,35 @@ func runCacheExporters(ctx context.Context, exporters []RemoteCacheExporter, j *
 	return cacheExporterResponse, nil
 }
 
-func exportWithInlineCacheExporter(ctx context.Context, e exporter.ExporterInstance, inlineExporter *RemoteCacheExporter, j *solver.Job, cached *result.Result[solver.CachedResult], inp *result.Result[cache.ImmutableRef]) (exporterResponse map[string]string, err error) {
-	if inlineExporter != nil {
-		if err := inBuilderContext(ctx, j, "preparing layers for inline cache", j.SessionID+"-cache-inline", func(ctx context.Context, _ session.Group) error {
-			if cached.Ref != nil {
-				dtic, err := inlineCache(ctx, inlineExporter.Exporter, cached.Ref, e.Config().Compression(), session.NewGroup(j.SessionID))
-				if err != nil {
-					return err
-				}
-				if dtic != nil {
-					inp.AddMeta(exptypes.ExporterInlineCache, dtic)
-				}
+func makeInlineCacheMetadata(ctx context.Context, e exporter.ExporterInstance, inlineExporter *RemoteCacheExporter, j *solver.Job, cached *result.Result[solver.CachedResult]) (map[string][]byte, error) {
```

Comment:
Not sure if this is the best name. This function actually does the cache export, blob creation, progress log. Making it sound like it just creates some metadata map isn't quite correct.